tests/aws/services/apigateway/test_apigateway_api.py

```diff
@@ -2013,6 +2014,91 @@ def test_gateway_response_crud(self, aws_client, apigw_create_rest_api, snapshot
 
         snapshot.match("get-deleted-gw-response", response)
 
+    @markers.aws.validated
+    @pytest.mark.skipif(
+        condition=not is_next_gen_api(), reason="Behaviour only present in next gen api"
+    )
+    def test_gateway_response_put(self, aws_client, apigw_create_rest_api, snapshot):
```

Comment:
nice new test 🆗 💯 very thorough, this is really awesome! thanks a lot for checking the missing behavior for `put_gateway_response`!